Puppetfile

```diff
@@ -23,11 +23,16 @@ mod 'puppetlabs-zone_core', '1.0.2'
 
 # Useful additional modules
 mod 'puppetlabs-package', '0.6.0'
-mod 'puppetlabs-puppet_conf', '0.3.1'
+mod 'puppetlabs-puppet_conf', '0.4.0'
 mod 'puppetlabs-python_task_helper', '0.3.0'
 mod 'puppetlabs-reboot', '2.2.0'
 mod 'puppetlabs-ruby_task_helper', '0.4.0'
 
+# Plugin modules
+mod 'puppetlabs-azure_inventory',
```

Comment:
I pushed 0.2.0 up manually, so we should be able to pull it from the forge now.

Reply:
nice, Updated.